iocore/cache/Cache.cc

```diff
@@ -1016,11 +990,7 @@ CacheProcessor::cacheInitialized()
             ram_cache_bytes += (int64_t)(http_ram_cache_size * factor);
             CACHE_VOL_SUM_DYN_STAT(cache_ram_cache_bytes_total_stat, (int64_t)(http_ram_cache_size * factor));
           } else {
-            factor = (double)(int64_t)(gvol[i]->len >> STORE_BLOCK_SHIFT) / (int64_t)theStreamCache->cache_size;
-            Debug("cache_init", "CacheProcessor::cacheInitialized - factor = %f", factor);
-            gvol[i]->ram_cache->init((int64_t)(stream_ram_cache_size * factor), vol);
-            ram_cache_bytes += (int64_t)(stream_ram_cache_size * factor);
-            CACHE_VOL_SUM_DYN_STAT(cache_ram_cache_bytes_total_stat, (int64_t)(stream_ram_cache_size * factor));
+            // ToDo: Assert? What's else here ?
```

Comment:
I would put a ink_assert() here, maybe an ink_release_assert()

Reply:
If you get there, your `gvol` array is messed up. Either assume it's not and don't check for `theCache`, or release assert as Bryan suggests.